src/main/java/com/univapay/sdk/types/brand/QrCpmBrand.java

```diff
@@ -1,72 +1,76 @@
 package com.univapay.sdk.types.brand;
 
-import com.google.gson.annotations.SerializedName;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
 
 public enum QrCpmBrand {
-  @SerializedName("qq")
-  QQ,
-  @SerializedName("pay_pay")
-  PayPay,
-  @SerializedName("bartong")
-  Bartong,
-  @SerializedName("we_chat")
-  WeChat,
-  @SerializedName("origami")
-  Origami,
-  @SerializedName("rakuten_pay")
-  RakutenPay,
-  @SerializedName("jkopay")
-  Jkopay,
-  @SerializedName("line_pay")
-  LinePay,
-  @SerializedName("au_pay")
-  AuPay,
-  @SerializedName("alipay_china")
-  AlipayCN,
-  @SerializedName("alipay_hk")
-  AlipayHK,
-  @SerializedName("alipay_singapore")
-  AlipaySG,
-  @SerializedName("kakaopay")
-  KakaoPay,
-  @SerializedName("tng")
-  TouchNGo,
-  @SerializedName("ezlink")
-  EzLink,
-  @SerializedName("gcash")
-  GCash,
-  @SerializedName("dana")
-  Dana,
-  @SerializedName("truemoney")
-  TrueMoney,
-  @SerializedName("d_barai")
-  DBarai,
-  @SerializedName("merpay")
-  Merpay,
-  @SerializedName("dash")
-  Dash,
-  @SerializedName("global_pay")
-  GlobalPay,
+  QQ("qq"),
 
-  @SerializedName("ginko_pay")
-  GinkoPay,
+  PayPay("pay_pay"),
+  Bartong("bartong"),
+  WeChat("we_chat"),
+  Origami("origami"),
+  RakutenPay("rakuten_pay"),
+  Jkopay("jkopay"),
+  LinePay("line_pay"),
+  AuPay("au_pay"),
 
-  @SerializedName("yucho_pay")
-  YuchoPay,
-  @SerializedName("hama_pay")
-  HamaPay,
-  @SerializedName("oki_pay")
-  OKIPay,
-  @SerializedName("yoka_pay_fukuoka")
-  YOKAPay_Fukuoka,
-  @SerializedName("yoka_pay_kumamoto")
-  YOKAPay_Kumamoto,
-  @SerializedName("yoka_pay_shinwa")
-  YOKAPay_Shinwa,
-  @SerializedName("hokuhoku_pay_hokkaido")
-  HokuHokuPay_Hokkaido,
-  @SerializedName("hokuhoku_pay_hokuriku")
-  HokuHokuPay_Hokuriku,
-  @SerializedName("coi_pay_hiroshima ")
-  CoiPay_Hiroshima
+  AlipayCN("alipay_china"),
+  AlipayHK("alipay_hk"),
+  AlipaySG("alipay_singapore"),
+  KakaoPay("kakaopay"),
+  TouchNGo("tng"),
+  EzLink("ezlink"),
+  GCash("gcash"),
+  Dana("dana"),
+  TrueMoney("truemoney"),
+
+  DBarai("d_barai"),
+
+  Merpay("merpay"),
+
+  Dash("dash"),
+  GlobalPay("global_pay"),
+
+  GinkoPay("ginko_pay"),
+  YuchoPay("yucho_pay"),
+  HamaPay("hama_pay"),
+  OKIPay("oki_pay"),
+  YOKAPay_Fukuoka("yoka_pay_fukuoka"),
+  YOKAPay_Kumamoto("yoka_pay_kumamoto"),
+  YOKAPay_Shinwa("yoka_pay_shinwa"),
+  HokuHokuPay_Hokkaido("hokuhoku_pay_hokkaido"),
+  HokuHokuPay_Hokuriku("hokuhoku_pay_hokuriku"),
+  CoiPay_Hiroshima("coi_pay_hiroshima");
```

Comment:
might as well add SMBC here now?

Reply:
I guess, would rather keep this separated for better visibility when cutting a release since the changelog is generated by the commits 

I will create a different PR with SMBC and merge back here 👍